src/WebJobs.Extensions.NotificationHubs/Config/NotificationHubsConfiguration.cs

```diff
@@ -43,6 +43,8 @@ public NotificationHubsConfiguration()
         /// </summary>
         public string HubName { get; set; }
 
+        internal TraceWriter Trace { get; set; }
```

Comment:
Yeah, we don't want consumers of this config class to have to worry about TraceWriters